### PR TITLE
FEATURE: add ability to customize "Reply" text

### PR DESCRIPTION
**Pull request by 3-w-c**
- New setting, `reply_button_text`
- Apply customised text to the composer button, the topic footer "Reply" button, and the post "Reply" button
- Add system tests for all cases

image

image

### CODE REVIEW

**Comment by 3-w-c** [comment]
@awesomerobot just wondering if you had chance to look at this, and if you had any advice? Thanks in advance! :pray:

**Comment by 3-w-c** [comment]
> Hi @3-w-c! thanks for giving this a shot, it mostly worked! There were some fringe cases I've covered as well.
> 
> I also ended up moving the repetitive `filteredSetting` getting to a utility and cleaned up a few things while I was at it.

That's amazing, thank you so much for the improvements!

> there's an API in Discourse for adding post menu buttons... I think this might be the way to go?

Will take a look, thanks.

**Comment by 3-w-c** [comment]
> Would you mind pulling my latest and checking to see if this is working how you'd expect it to?

It's working beautifully, very well done @awesomerobot :clap: 

The last "nice to have" would be customising the button for logged-out users, but I guess we could always come back to that afterwards?

**Comment by 3-w-c** [comment]
Thanks @awesomerobot, makes sense. Let me know if there's anything I can do to help :pray:

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
```
Comment (nbianca) [inline .vscode/settings.json:1]: I think this should not be committed.

**Review by nbianca — APPROVED** [review]

**Pull request merged**